### ISSUE TITLE
Implement learning rate decay

### ISSUE THREAD

**Issue by jonathat22** [jonathat22/Brain-Tumor-Classification#11]
By gradually decreasing the learning rate over the course of gradient descent, we can prevent the model from missing the minimum
